scripts/backlog-1105/update-by-uuid.sh

```diff
@@ -0,0 +1,40 @@
+#! /bin/bash
+
+# We are given a xml file and extract the geonetwork uuids from it to find records for updating
```

Comment:
Why not a generic script we can use again?
The file input should be a list of UUID's produced by other means, such as https://github.com/aodn/utilities/blob/master/geonetwork/gn-tool/get-uuids-from-file.py

Reply:
It could even be done in mafia java land?

Reply:
The uuid list and transform xsl files are now parameters